backend/tests/integration/editor/test_editor_views.py

```diff
@@ -164,15 +165,59 @@ def test_create_rt_ticket_returns_403_on_authentication_error(inspire_app):
 
 @patch("inspirehep.editor.views.tickets")
 def test_resolve_rt_ticket(mock_tickets, inspire_app):
-    mock_tickets.create_ticket.return_value = 1
-    mock_tickets.get_rt_link_for_ticket.return_value = "http://rt_address"
+    mock_tickets.get_rt_user_by_email.side_effect = NoUsersFound
```

Comment:
Why `NoUsersFound`?

Reply:
As this is the old test I'm mocking methods in a way for a function to run as it was running before my changes.